_docs/style/typography.md

```diff
@@ -29,61 +23,167 @@ body text. Hierarchy can be established using size and
+{% column left:"hxCol-8" %}
+### Body Styles
+{: .hxSubSectionTitle }
+{% endcolumn %}
 </div>
 
 <div class="hxRow">
-{% include type-styles.html htmlexample='<h1 class="headerDisplayList header-alt">Light Heading</h1>' name="H1 Light" typeface="Roboto" weight="Thin (100)" size="24px / 1.5em" csscode="Grey 900" hex="#424242" bottomspace="24px / 1.5em" %}
-{% include type-styles.html htmlexample='<h2 class="headerDisplayList">Subheading</h2>' name="H2" typeface="Roboto" weight="Regular (400)" size="20px / 1.25em" csscode="Grey 900" hex="#424242" bottomspace="12px / 0.75em" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Body Text
+{: .hxContainerTitle }
+Body text is the default style and should be used across the interface for most information.
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% include type-styles.html htmlexample='<p>Body Text</p>' typeface="Roboto" weight="Regular (400)" fontsize="16px / 1rem" csscode="Gray 900" hex="#424242" bottomspace="24px / 1rem" linespace="1.5" %}
+{% endcolumn %}
 </div>
 
 <div class="hxRow">
-{% include type-styles.html htmlexample='<h3 class="headerDisplayList">Subheading</h3>' name="H3" typeface="Roboto" weight="Medium (500)" size="16px / 1.25em" csscode="Grey 900" hex="#424242" bottomspace="12px / 0.75em" %}
-{% include type-styles.html htmlexample='<span class="body1 headerDisplayList">Body 1</span>' name="Body" typeface="Roboto" weight="Regular (400)" size="16px / 1em" csscode="Grey 900" hex="#424242" linespace="24px / 1.5em" bottomspace="24px/1.5em" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Subdued Style
+{: .hxContainerTitle }
+Subdued style can be applied to any text 14px and taller, and is used to let more important text stand out. It is not the same as an inactive link or button.
+
+*Do not subdue text on backgrounds darker than Gray 25 (#fafafa) or it will not pass accessibiity standards.*
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% include type-styles.html htmlexample='<p class="hxSubdued">Subdued Style</p>' typeface="Roboto" csscode="Gray 900" hex="#424242" %}
+{% endcolumn %}
 </div>
 
+
 <div class="hxRow">
-{% include type-styles.html htmlexample='<a>Link</a>' name="Link" typeface="Roboto" weight="Regular (400)" size="16px / 1em" csscode="Cyan 900" hex="#OC7C84" linespace="24px / 1.5em" %}
-{% include type-styles.html htmlexample='<span class="body2 headerDisplayList">Body 2</span>' name="Body 2" typeface="Roboto" weight="Regular (400)" size="14px / .875em" csscode="Grey 900" hex="#424242" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Sub-Body Text
+{: .hxContainerTitle }
+Sub-body text should be used sparingly as it is the least readable size. We make it available for use cases when smaller text is appropriate, such as icon labels.
```

Comment:
Use sub-body text sparingly, since it is the least readable size. You can use it when smaller text is appropriate, such as for icon labels.